packages/react-router/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "scripts": {
     "build": "npm run clean && npm run compile",
+    "build.watch": "npm run compile -- --watch",
```

Comment:
Should the new `build.watch` commands include the other steps from the original `build`s? For example, the `build` here also includes `npm run clean`.

Reply:
The `--watch` flag only works for compilation not other commands. You could have `npm run clean` run at the beginning, but it wouldn't re-run every time there was a change.